.github/workflows/generate.yml

```diff
@@ -0,0 +1,37 @@
+name: Generate extension list
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch: ~
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      FILE: extensions.json
+
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Build JSON file
+        run: php ./generate.php
```

Comment:
Maybe make sure that the main errors will return a non-zero error code, so that the workflow will abort

https://github.com/FreshRSS/Extensions/blob/93c042a57d7f45d7b8aafc71cdffe689998152f9/generate.php#L14

Reply:
Good point. I'll add something to handle that better.

Reply:
@Alkarex I've refactor the code to leverage PHP 7.3 features. As it should run on PHP 8.1 in the container, we should be fine.